app/views/docs/authentication.phtml

```diff
@@ -0,0 +1,281 @@
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<p>
+    After an account is created, the newly created account must be verified through the <a href="/docs/account#accountCreateVerification">account verification route</a> provided by the Appwrite Accounts API, after which the user can login to their new account.
+</p>
+
+<h3><a href="/docs/authentication#anonymous-user" id="anonymous-user">Anonymous User</a></h3>
+
+<p>
+    Anonymous authentication allows users of your application to create a temporary valid session without creating an account.
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+const promise = account.createAnonymousSession();
+
+promise.then(function (response) {
+    console.log(response); // Success
+}, function (error) {
+    console.log(error); // Failure
+});
+
+</code></pre>
+</div>
+
+<h3><a href="/docs/authentication#oauth" id="oauth">OAuth</a></h3>
+
+<p>
+    OAuth is another way to authenticate a user using a multi-step process. When using OAuth to authenticate, the request to authenticate is initiated from the client application, then the user is redirected to an OAuth2 provider to complete the authentication step, and finally, the user is redirected back to the client application if completed successfully. This provides integration with many third-party services that provide their own OAuth integration as a more secure approach than providing a username/password directly.
+</p>
+
+<p>
+    The example below shows you how to authenticate with OAuth2 using the Web SDK and Amazon's OAuth system.
+</p>
+
+<div class="ide" data-lang="javascript" data-lang-label="Web SDK">
+    <pre class="line-numbers"><code class="prism language-javascript" data-prism>import { Client, Account } from "appwrite";
+
+const client = new Client();
+
+const account = new Account(client);
+
+client
+    .setEndpoint('https://[HOSTNAME_OR_IP]/v1') // Your API Endpoint
+    .setProject('5df5acd0d48c2') // Your project ID
+;
+
+// Go to OAuth provider login page
+account.createOAuth2Session('amazon');
+
+</code></pre>
+</div>
+
+<p>  
+    If there is already an active session, the new session will be attached to the logged-in account. If there are no active sessions, the server will attempt to look for an account with the same email address as the email received from the OAuth2 provider and attach the new session to the existing Account. If no matching account is found - the server will create a new account.
```

Comment:
`If there is already an active session, the new session will be attached to the logged-in account.`

This to me sounds like "If you are already logged in with email&password, and you try to login with oauth, these two will now connect to the same account. Not sure if that is true.